comments/comments-router.js

```diff
@@ -29,6 +60,7 @@ router.post("/", restricted, async (req, res) => {
         res.status(500).json({
             error
         });
+        k
```

Comment:
whats this k ?

Reply:
So...I'm currently using a gaming mouse that has keyboard keybindings on it. Every time I scroll, it drops a "k" randomly. My bad. I need a new standard mouse.